lib/attrsets.nix

```
@@ -444,6 +444,105 @@ rec {
     else
       [];
 
+  /* Recursively collect values produced by function `f` that verify a given
```

Please change the format to commonmark. This PR is open for so long... conventions beeing changed in master branch already.

https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix#L65

```suggestion
  /** Recursively collect values produced by function `f` that verify a given
```